### ISSUE TITLE
Discord server requires phone number now.

### ISSUE THREAD

**Issue by kspeakman** [mrhid6/SF_Mod_RefinedPower#68]
Is that on purpose? I am not giving discord my phone number. I get enough spam calls as it is.

**Comment by Imachicken29** [comment]
Yes it's on purpose, it's 2FA, i highly suggest turning on 2FA for your own security. Discord also won't send spam calls but just emails on a half-year basis about stuff and you can unsubscribe from that stuff as well. Please msg the admins first and not making a bug report on the discord for the mod

**Comment by kspeakman** [comment]
@Imachicken29 I apologize for contacting you in an undesirable way, and I will not be doing so in the future to respect your wishes. I though it was fair game as it represents a project-level configuration change from the last time I joined the discord server. And I basically have no other way to contact you, as I try to limit the number of accounts which data mine and sell me to advertisers. I will be deleting my discord after this as well, as I only made it to participate in Satisfactory mod conversations.

I do really appreciate the mod and your time spent on it. Best wishes!
